types/block.go

```diff
@@ -1175,9 +1100,9 @@ func (ec *ExtendedCommit) BitArray() *bits.BitArray {
 	if ec.bitArray == nil {
 		ec.bitArray = bits.NewBitArray(len(ec.ExtendedSignatures))
 		for i, extCommitSig := range ec.ExtendedSignatures {
-			// TODO: need to check the BlockID otherwise we could be counting conflicts,
-			// not just the one with +2/3 !
-			ec.bitArray.SetIndex(i, !extCommitSig.Absent())
+			// TODO(sergio): need to check the BlockID otherwise we could be counting conflicts,
```

Comment:
Can you please remove this TODO? I don't even recall what it was about (so not sure it is pertinent)